docs/examples/plotting/How-to-use-Plottr-with-QCoDeS-for-live-plotting.ipynb

```diff
@@ -0,0 +1,206 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to use Plottr with QCoDeS for live plotting?\n",
+    "\n",
+    "Note: This notebook assumes that the user has a `conda` environment on their system with latest versions of `plottr` installed. If you do not have plottr installed, please use command `pip install plottr` for installing it into your environment.\n",
```

Comment:
perhaps rather refer to plottr's readme (or docs?) for installation instructions becasue qcodes shouldnb't need to maintain those

Reply:
Sure, will do.